apps/air-discount-scheme/web/screens/Admin/consts.ts

```diff
@@ -36,4 +36,5 @@ export const airlineOptions = [
 
 export type FilterInput = FlightLegsInput & {
   airline: { value: string }
+  isExplicit: string
```

Comment:
Why not `boolean`?

Reply:
The `<Checkbox>` controller from which this input arrives does not allow boolean as values